assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -25,43 +25,46 @@ import { __, sprintf } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
+import { Headline, Text, THEME_CONSTANTS } from '../../../design-system';
 import { STORY_STATUS } from '../../constants';
 import { titleFormatted } from '../../utils';
 import { DashboardStatusesPropType } from '../../types';
-import { Paragraph2 } from '../typography';
 import InlineInputForm from '../inlineInputForm';
-import { Link } from '../link';
 
 const StyledCardTitle = styled.div`
   padding-top: 12px;
   display: inline-block;
-  overflow: hidden;
+  max-width: calc(100% - 20px);
```

Comment:
Updated how card space is handled to allow story title to have keyboard focus style not get cut off.